Climate_police/tests/hw7-mao/us_state_abbrev.py

```diff
@@ -0,0 +1,52 @@
+us_state_abbrev = {
```

Comment:
For choropleth map, there's build-in support for the U.S. states, which actually makes sense because there are too many cities, so it's difficult to use choropleth map. Bubble map (or heat map) might be a better option for cities.